### DISCUSSION TITLE
How to set window title from "predefined task"

### DISCUSSION THREAD

**Discussion by spotlight2001** [cmderdev/cmder#2662] [Q&A]
I am working on several projects. For each project I use a cmder window, with several tabs. Usually I click on the wrong opened cmder in the taskbar. How can I set the window title, which I expect to show up in the windows taskbar.

**Comment by daxgames** [comment]
Not really a Cmder question more related to Conemu.  This might be what you want?

https://conemu.github.io/en/TabVsTitle.html
